pkg/cas/blob_access_content_addressable_storage.go

```diff
@@ -87,6 +88,10 @@ func (cas *blobAccessContentAddressableStorage) GetFile(ctx context.Context, dig
 		directory.Remove(name)
 		return err
 	}
+	time := time.Date(2011, time.November, 11, 11, 11, 11, 0, time.UTC)
```

Comment:
I have some other code that I want to push out at some point in the future that also has to set file timestamps to a deterministic value. What are your thoughts on adding a constant to pkg/filesystem that we can use throughout the codebase?

```go
var (
    DeterministicFileModificationTimestamp = time.Date(2000, time.January, 1, 0, 0, 0, 0, time.UTC)
)
```

I can adjust my patchset to make use of that constant as well. Silly question: we already started using 2000-01-01 on our end. Would you mind using that as well?